### PR TITLE
OperationalInsights Management lib from swagger and tests for swagger

### PR DESCRIPTION
**Pull request by xizha162**
The first version of OperationalInsights .NET SDK generated from swagger using AutoRest plus the tests for the new SDK.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@xizhamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (xizha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/DataSourcesOperations.cs [hunk 1]
```diff
@@ -0,0 +1,983 @@
...
+    using Microsoft.Rest.Serialization;
+    using Newtonsoft.Json;
+    using Microsoft.Rest.Azure.OData;
+    using Microsoft.Rest.Azure;
+    using Models;
+
+    /// <summary>
+    /// DataSourcesOperations operations.
+    /// </summary>
+    internal partial class DataSourcesOperations : IServiceOperations<OperationalInsightsManagementClient>, IDataSourcesOperations
```
Comment (markcowl) [inline src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/DataSourcesOperations.cs]: This should be DataSourceOperations, by naming convnetion

**Review by markcowl — CHANGES_REQUESTED** [review]
It looks like your swagger spec is not checked in, please include the PR to your swagger spec.  Also, please note the naming for operations classes, this applies throughout

DataSourcesOperations -> DataSOurceOperations.

Finally, you should refer to your swagger spec in generate.cmd through the commit, and not through the floating branch it is in, once it is checked in.

**Comment by markcowl** [comment]
@xizhamsft Please respond.  We will close this PR early next week if there are no changes or comments.

**Comment by cormacpayne** [comment]
@xizhamsft closing PR as there has been no activity for two weeks. Please re-open the PR when comments can be addressed.

**Pull request closed without merging**